From 3ec76f32da82973a39372b8662c130342c6df45e Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:12:17 -0400
Subject: [alsa] hiface: Fix typo in 352800 rate definition

Message-id: <140972823573-11-patchset-jkysela@redhat.com>
Patchwork-id: 90711
O-Subject: [RHEL 7.1 PATCH 010/442] ALSA: hiface: Fix typo in 352800 rate definition
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The Vaughan device support the 352800 rate and not
the 352000

Signed-off-by: Michael Trimarchi <michael@amarulasolutions.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 150116bcfbd9004f01e682535baddaca2b90d3b2)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/usb/hiface/pcm.c b/sound/usb/hiface/pcm.c
index c21a3df..2c44139 100644
--- a/sound/usb/hiface/pcm.c
+++ b/sound/usb/hiface/pcm.c
@@ -110,7 +110,7 @@ static const struct snd_pcm_hardware pcm_hw = {
 #define HIFACE_RATE_96000  0x4a
 #define HIFACE_RATE_176400 0x40
 #define HIFACE_RATE_192000 0x48
-#define HIFACE_RATE_352000 0x58
+#define HIFACE_RATE_352800 0x58
 #define HIFACE_RATE_384000 0x68
 
 static int hiface_pcm_set_rate(struct pcm_runtime *rt, unsigned int rate)
@@ -141,8 +141,8 @@ static int hiface_pcm_set_rate(struct pcm_runtime *rt, unsigned int rate)
  case 192000:
   rate_value = HIFACE_RATE_192000;
   break;
- case 352000:
-  rate_value = HIFACE_RATE_352000;
+ case 352800:
+  rate_value = HIFACE_RATE_352800;
   break;
  case 384000:
   rate_value = HIFACE_RATE_384000;
-- 
1.7.1